cli-feed-v4.json

```diff
@@ -2100,6 +2105,306 @@
+          "sha2": "a6137a79a14a5bd55d7fafe3c05200db7c767c9ca0e17cbca9f5d24c273100ea",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.3528/Azure.Functions.Cli.min.win-x64.3.0.3528.zip",
+          "sha2": "2528ea9cee707cc4787f201e1705b4264cc2b8b8d67a3fabdfad3ca1b5a06da4",
+          "size": "minified",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.3528/Azure.Functions.Cli.win-x86.3.0.3528.zip",
+          "sha2": "83478b84d66ed52e508e12a926666e1ad85fb9b7e7e091820b52f7ae3aae5bcd",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.3528/Azure.Functions.Cli.min.win-x86.3.0.3528.zip",
+          "sha2": "802e2ca6a355099ea7491beec932f80ae6fce8968b8f94a40cbb4dd36c3490fc",
+          "size": "minified",
+          "default": "true"
+        }
+      ]
+    },
+    "4.0.0": {
+      "templates": "https://functionscdn.azureedge.net/public/TemplatesApi/3.1.1648.zip",
+      "workerRuntimes": {
+        "dotnet": {
+          "net6": {
+            "displayInfo": {
+              "displayName": ".NET 6",
+              "hidden": false,
+              "displayVersion": "v4",
+              "targetFramework": ".NET",
+              "description": "Preview"
+            },
+            "capabilities": "net6",
+            "sdk": {
+              "name": "Microsoft.NET.Sdk.Functions",
+              "version": "3.0.13"
+            },
+            "default": true,
+            "localEntryPoint": "func.exe",
+            "targetFramework": "netcore6.0",
+            "itemTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ItemTemplates/4.0.1844-preview1",
+            "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ProjectTemplates/4.0.1844-preview1",
+            "projectTemplateId": {
+              "csharp": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.3.x"
+            },
+            "localContainerBaseImage": "DOCKER|mcr.microsoft.com/azure-functions/dotnet:6.0-appservice",
+            "serviceAppSettings": {
+              "FUNCTIONS_EXTENSION_VERSION": "~4",
+              "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+            },
+            "windowsSiteConfig": {
+              "netFrameworkVersion": "v6.0"
+            },
+            "linuxSiteConfig": {
+              "linuxFxVersion": "DOTNET|6.0"
+            }
+          },
+          "net5-isolated": {
+            "displayInfo": {
+              "displayName": ".NET 5",
```

Comment:
I'm not the expert here -- but I thought these were supposed to be ".NET 5.0" and ".NET 6.0" (as opposed to "5" and "6").

Reply:
That makes sense. @phil-allen-msft, ok to make this change in the display name?